File: online-judge-solutions/Codeforces/412B.py
#https://codeforces.com/problemset/problem/412/B
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
n,k=map(int,input().split())
a=list(map(int,input().split()))
a.sort(reverse=True)
print(a[k-1])

File: online-judge-solutions/Codeforces/412B.cpp
//https://codeforces.com/problemset/problem/412/B
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	0 KB
#include<iostream>
#include<algorithm>
using namespace std;
int main(){
	int n,k;
	while(cin>>n>>k){
		int a[n];
		for(int i=0;i<n;i++){cin>>a[i];}
		sort(a,a+n,greater<int>());
		cout<<a[k-1]<<endl;
	}
	return 0;
}
